jina/docker/hubio.py

```diff
@@ -178,6 +178,36 @@ def login(self):
             # use default login
             self._client.login(**get_default_login(), registry=self.args.registry)
 
+    def list(self, name: str = None, kind: str = None, type: str = 'pod', keywords: list = ["numeric", "sklearn"]):
+        """Lists executors allowing filters for name, kind, type and keywords """
+        if not is_db_envs_set():
+            self.logger.error('DB environment variables are not set! aborting list command.')
+            return
+
+        if kind:
+            kind_query = {'manifest_info.kind': kind}
+            sub_query = f'{kind_query}'
+        if type:
+            type_query = {'manifest_info.type': type}
+            sub_query = f'{sub_query}, {type_query}'
```

Comment:
If the 1st condition fails, `{sub_query}` wouldn't be defined, will raise an error here.

Reply:
yes, corrected it.